### PR TITLE
Added many new pages for things like mpd, ssh, printing, wget, ffmpeg, and pandoc.

### PR DESCRIPTION
**Pull request by daschwa**
I have added pages for using imagemagick, mpc, printing, converting source code to pdf, spoofing MAC addresses, setting up ssh, ed, git submodules, pandoc, wget, and ffmpeg.pdf, Hopefully some useful stuff for other people.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (sloria) [inline .gitignore]: I'd rather not add a gitignore for OS- and editor-generated files. These belong in a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: use-and-setup-ssh.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+title: settup and use SSH
```
Comment (sloria) [inline use-and-setup-ssh.md]: Small typo here. "setup"

File: usefull-ffmpeg.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+# Add Audio track to Video
+```
+ffmpeg -i input.mkv -i input.mp3 -c copy -shortest -map 0:0 -map 1:0 output.mkv
+```
+
+# Boost Audio
+Boost audio by 150%
+'''
```
Comment (sloria) [inline usefull-ffmpeg.md]: These should be triple-backticks, not quotes.

**Comment by sloria** [comment]
I've merged and made a few small edits for spelling and formatting. Thanks a lot for this big contribution!

**Comment by daschwa** [comment]
Thanks! Great catches too.

**Pull request merged**